### PR TITLE
feat: added solana to flask build

### PR DESCRIPTION
**Pull request by javiergarciavera**
## **Description**

Added Solana feature on Flask build

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/29195?quickstart=1)

## **Related issues**

N/A

## **Manual testing steps**

- Open Experimental settings and enable Solana
- You should see Solana option when creating account

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by aganglada — DISMISSED** [review]

**Comment by aganglada** [comment]
@javiergarciavera Do we add the `solana` feature to flask? or do we just remove `solana` and use `build-flask`

**Comment by javiergarciavera** [comment]
> @javiergarciavera Do we add the `solana` feature to flask? or do we just remove `solana` and use `build-flask`

I guess that's a question that needs to be agreed from the team?

**Comment by ccharly** [comment]
> @javiergarciavera Do we add the `solana` feature to flask? or do we just remove `solana` and use `build-flask`

IMO, we can keep going with the `solana` feature for now. If for whatever reason we decide to revert this PR, it will be easier to revert it.

We can always go with `build-flask` later one if needed, WDYT?

File: privacy-snapshot.json [hunk 1]
```diff
@@ -46,17 +46,18 @@
   "on-ramp-content.uat-api.cx.metamask.io",
   "phishing-detection.api.cx.metamask.io",
   "portfolio.metamask.io",
+  "price-api.metamask-institutional.io",
```
Comment (ccharly) [inline privacy-snapshot.json:49]: @aganglada is this new host expected? Are you guys making any calls to that API on your Snap?

Comment (aganglada) [reply privacy-snapshot.json:49]: We are using MMI price api in order to get rates from solana right now, this is temporary tho as we will be using platform price api as a final solution.
https://github.com/MetaMask/snap-solana-wallet/blob/main/packages/snap/src/core/clients/price-api/price-api-client.ts

**Review by aganglada — APPROVED** [review]

**Review by pedronfigueiredo — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]
Privacy snapshot changes LGTM; the price API added here is only used in a Snap, so it would not be called with basic functionalities disabled.

**Pull request merged**